dsp-worker.js

```
@@ -307,32 +326,49 @@ async function runPipeline(msg) {
+    }
+
     // ===== PASS 10: EXPORT (S35–S36) =====
     // S35–S36 handled by caller (WAV encode / video mux)
     progress(36, 100, 'Pipeline Complete');
 
+    // [C18] Inline calcPeak/calcRMS (methods not on DSP instance)
+    const peak = outputData.reduce((m, v) => Math.max(m, Math.abs(v)), 0);
+    const rms = Math.sqrt(outputData.reduce((s, v) => s + v * v, 0) / outputData.length);
+
     // Return processed data (Transferable = zero-copy)
     self.postMessage({
```

ESLint

[error] 'self' is not defined.

(no-undef)